pkgs/by-name/cu/cubeb/package.nix

```diff
@@ -58,12 +62,27 @@ stdenv.mkDerivation {
     backendLibs = lib.optionals lazyLoad backendLibs;
   };
 
+  postInstall = ''
+    # TODO: remove after https://github.com/mozilla/cubeb/pull/813 is merged
+    mkdir -p $out/lib/pkgconfig/
+    echo > $out/lib/pkgconfig/libcubeb.pc \
+    "Name: libcubeb
+    Description: Cross platform audio library
+    Version: 0.0.0
+    Requires.private: libpulse
+    Libs: -L"$out/lib" -lcubeb
+    Libs.private: -lstdc++"
+  '';
+
```

Comment:
I wonder whether it wouldn't be better to just vendor the patch that you submitted upstream, so that the .pc file is correctly generated?
Not blocking for this PR though.